Dockerfile-backup-driver

```diff
@@ -12,13 +12,8 @@
 # See the License for the specific language governing permissions and
 # limitations under the License.
 
-FROM ubuntu:focal
-RUN apt-get update && \
-    apt-get install -y --no-install-recommends ca-certificates wget bzip2 && \
-    apt-get remove -y wget bzip2 && \
-    rm -rf /var/lib/apt/lists/*
-RUN mkdir /vddkLibs
+FROM gcr.io/distroless/base-debian10:nonroot
 COPY /bin/linux/amd64/lib/vmware-vix-disklib/lib64/* /vddkLibs/
 ADD /bin/linux/amd64/backup-driver* /backup-driver
-USER nobody:nogroup
+USER nonroot:nonroot
```

Comment:
Please make sure this is synchronized with the Velero container configuration.  We had problems in the past with nobody/nogroup vs nobody/nobody